### DISCUSSION TITLE
I can only read. I can't write to the Nefit envilne full electric of 2021

### DISCUSSION THREAD

**Discussion by RR13466** [emsesp/EMS-ESP32#1016] [General]
I can only read. I can't write to the Nefit envilne full electric of 2021. All on the webbase.

**Comment by proddy** [comment]
Oh. Let me find my crystal ball 😇

**Comment by proddy** [comment]
got any more info for us?

**Comment by RR13466** [comment]
I have a Gateway E32 (Ethernet + WiFi Edition), from bbqKees, connect with a Nefit full electric boiler. Connect on wifi. The software on the gateway is ems-esp 3.5.0b9 I can monitor everyving but i can't change anything The boiler is from end 2021. The older software firmware didn't change anything. i hoop you can help me. If you need more information ask me.

**Comment by proddy** [comment]
See https://emsesp.github.io/docs/#/Support
